rpa-manager/modules/ROOT/pages/processautomation-prepare-project-category.adoc

```diff
@@ -0,0 +1,77 @@
+= Manage Categories
+
+Process team members organize projects into categories. 
+
+Manage a set of categories from which process team members can choose one when they create a xref:processevaluation-propose.adoc#create-a-process-evaluation[process evaluation] or an xref:processautomation-manage.adoc#create-an-automation-project[automation project].
```

Comment:
```suggestion
Manage a set of categories from which process team members can choose when they create a xref:processevaluation-propose.adoc#create-a-process-evaluation[process evaluation] or an xref:processautomation-manage.adoc#create-an-automation-project[automation project].
```

Reply:
They only can choose one (and not several, unfortunately)

Reply:
OK! (Ignore comment)